### ISSUE TITLE
invalid context 0x0

### ISSUE THREAD

**Issue by paulz** [ashfurrow/Nimble-Snapshots#41]
Thank you for useful cocoapod!
When running tests we are getting:

```
Feb 23 01:55:29  Example[69449] <Error>: CGContextSaveGState: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextSaveGState: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextSetBlendMode: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextSetAlpha: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextTranslateCTM: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextScaleCTM: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextDrawImage: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextRestoreGState: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Feb 23 01:55:29  Example[69449] <Error>: CGContextRestoreGState: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
```

setting symbolic breakpoint at `CGPostError`
shows the offending call to render view without context

Would it be possible to make it optional to pretender views without contexts?
Seems like it was added in this change: https://github.com/ashfurrow/Nimble-Snapshots/pull/39 https://github.com/ashfurrow/Nimble-Snapshots/commit/1f3cf29237e43478545970607c96f92399e38f62 to fix https://github.com/ashfurrow/Nimble-Snapshots/issues/38

**Comment by ashfurrow** [comment]
Makes sense. I get those too but ignore them, had to add the code to render it to get everything to render correctly. No harm in making it optional. 
-- 
Ash Furrow
https://ashfurrow.com/

**Comment by orta** [comment]
Fixed in #47
